Task03

gemansipator opened the pull request:
1. Создать массив на 10 строк.
 2. Ввести с клавиатуры 8 строк и сохранить их в массив.
 3. Вывести содержимое всего массива (10 элементов) на экран в обратном порядке.
 Каждый элемент - с новой строки.

gemansipator committed:
Task03 1. Создать массив на 10 строк.
        2. Ввести с клавиатуры 8 строк и сохранить их в массив.
        3. Вывести содержимое всего массива (10 элементов) на экран в обратном порядке.
        Каждый элемент - с новой строки.

RinatBeybutov commented on src/main/java/site/javadev/lesson_6/homework/Task03.java:
```diff
@@ -0,0 +1,22 @@
+        for (int i = 0; i < 8; i++) {
+            str[i] = scanner.next();
+        }
+        for (int i = 9; i < str.length ; i--) {
```
здесь нужно переписать условие окончания цикла, а if удалить

gemansipator committed:
Task03 1. Создать массив на 10 строк.
        2. Ввести с клавиатуры 8 строк и сохранить их в массив.
        3. Вывести содержимое всего массива (10 элементов) на экран в обратном порядке.
        Каждый элемент - с новой строки.

RinatBeybutov commented:
принято
